src/status_im/keycard/login.cljs

```diff
@@ -70,7 +70,7 @@
                                    :status     nil))
       :hide-popover nil})
    (when (:multiaccount db)
-     (navigation/change-tab :profile))
+     (navigation/change-tab :browser-stack)) ;; Profile tab - Currently browser tab is used for profile
```

Comment:
could you pls elaborate?

Reply:
Hi @flexsurfer,
Thank you for reviewing PR.
Currently, we don't have a separate screen for the profile, and just using the 4th bottom tab(browser stack) as a container for it.
So if we need to navigate to the `:profile` tab we have to navigate to `:browser-stack` instead. Once screen will be ready, we can change this to :navigate-to `:profile`